dockstore-client/src/main/java/io/dockstore/client/cli/nested/CromwellLauncher.java

```diff
@@ -49,6 +52,10 @@ public CromwellLauncher(AbstractEntryClient abstractEntryClient, DescriptorLangu
     public void initialize() {
         // initialize cromwell location from ~/.dockstore/config
         INIConfiguration config = Utilities.parseConfig(abstractEntryClient.getConfigFile());
+        cromwellVmOptions = (List)(config.getList("cromwell-vm-options"));
```

Comment:
If cromwell-vm-options does not exist in the config file, won't this return null? If so, then your code on line 94 will fail with an NPE.

Reply:
Does not return null if no cromwell-vm-options in config; but returns an ArrayList of size= 0.
So code is OK